test/cases/diff-manifests.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Colorful output.
+function greenprint {
+    echo -e "\033[1;32m[$(date -Isecond)] ${1}\033[0m"
+}
+function redprint {
+    echo -e "\033[1;31m[$(date -Isecond)] ${1}\033[0m"
+}
+
+greenprint "Fetching origin/main"
+git fetch origin main
+
+greenprint "Getting revision IDs for HEAD and merge-base"
+head=$(git rev-parse HEAD)
+mergebase=$(git merge-base HEAD origin/main)
```

Comment:
This will be weird when we make PR to rhel-* branches. Not 100% sure if it's worth solving and how to solve it.

Reply:
Oh yeah, didn't think of that.  Can we pass the target branch name from GitHub to GitLab CI and use that instead?

Reply:
Alternatively, we can probably query GH for the PR info and get the target branch name from there.

Reply:
Updated.  Gets the base branch from GH.  Also, it skips the run if the `$CI_COMMIT_BRANCH` doesn't match `PR-*`.